backend/ordd_api/models.py

```diff
@@ -87,11 +87,28 @@ def __str__(self):
         return self.name
 
 
-class KeyDatasetNameManager(models.Manager):
+class KeyPerilObsoleteManager(models.Manager):
     def get_by_natural_key(self, name):
         return self.get(name=name)
 
 
+class KeyPerilObsolete(models.Model):
+    objects = KeyPerilObsoleteManager()
+
+    name = models.CharField(max_length=32, blank=False, unique=True)
+
+    def natural_key(self):
+        return [self.name]
+
+    def __str__(self):
+        return self.name
+
+
+class KeyDatasetNameManager(models.Manager):
+    def get_by_natural_key(self, name, category):
+        return self.get(name=name, category=category)
+
+
```

Comment:
It was a way to be able to go backward with migrations (in the other case the table content was missed).